src/Hilke.KineticConvolution/Helpers/SvgBuilder.cs

```diff
@@ -0,0 +1,203 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+using Hilke.KineticConvolution.DoubleAlgebraicNumber;
+
+namespace Hilke.KineticConvolution.Helpers
+{
+    public class SvgWriter
+    {
+        private const double DefaultMinimumArcRadius = 1.0e-02;
+        private const double DefaultLineWidth = DefaultMinimumArcRadius;
+        private const double DefaultSvgScaling = 1000.0;
+
+        private readonly ConvolutionFactory<double> _factory;
+        private readonly double _minimumArcRadius, _lineWidth, _svgScaling;
+        private readonly List<(IReadOnlyList<Tracing<double>> Tracings, string Name)> _manyTracings;
+
+        public SvgWriter(
+            double minimumArcRadius = DefaultMinimumArcRadius,
+            double lineWidth = DefaultLineWidth,
+            double svgScaling = DefaultSvgScaling)
+        {
+            _factory = new ConvolutionFactory<double>(new DoubleAlgebraicNumberCalculator());
```

Comment:
It's a bit odd that we need a tracing factory to emit an svg.. Can't we do without that? Without creating intermediate tracings, and directly emitting the result?